src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,21 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { TodoWithUser } from '../../react-app-env';
+
+type Props = {
+  todos: TodoWithUser[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <React.Fragment key={todo.id}>
+          <TodoInfo
+            todo={todo}
+          />
```

Comment:
Why did you add React.Fragment. I think it will be better to add a key in the TodoInfo component.

Reply:
Just forgot that we can do like this